src/Umbraco.Infrastructure/Intall/InstallHelper.cs

```diff
@@ -107,19 +111,7 @@ public async Task InstallStatus(bool isCompleted, string errorMsg)
 
         internal static string GetDbProviderString(ISqlContext sqlContext)
         {
-            var dbProvider = string.Empty;
-
-            // we don't have DatabaseProvider anymore...
-            //dbProvider = ApplicationContext.Current.DatabaseContext.DatabaseProvider.ToString();
-            //
-            // doing it differently
-            var syntax = sqlContext.SqlSyntax;
-            if (syntax is SqlCeSyntaxProvider)
-                dbProvider = "SqlServerCE";
-            else if (syntax is SqlServerSyntaxProvider)
-                dbProvider = (syntax as SqlServerSyntaxProvider).ServerVersion.IsAzure ? "SqlAzure" : "SqlServer";
-
-            return dbProvider;
+            return sqlContext.SqlSyntax.DbProvider;
```

Comment:
Maybe we don't even need the method `GetDbProviderString` anymore and instead just use SqlSyntax to resolve this info?